pkg/utils/args/args_test.go

```diff
@@ -210,6 +211,43 @@ var _ = Describe("ParseArguments", func() {
 
 	})
 
+	Context("ClusterIdentity", func() {
+
+		type parseClusterIdentityTestCase struct {
+			args          []string
+			expectedError OmegaMatcher
+		}
+
+		DescribeTable("CIDR table",
+			func(c parseClusterIdentityTestCase) {
+				flagset := flag.NewFlagSet("test", flag.ContinueOnError)
+				flags := NewClusterIdentityFlagsFor("cluster", "The %s of the current cluster", flagset)
+				Expect(flagset.Parse(c.args)).To(Succeed())
+				_, err := flags.Read()
+				Expect(err).To(c.expectedError)
+			},
+
+			Entry("no cluster ID", parseClusterIdentityTestCase{
+				args:          []string{"--cluster-name=foo"},
+				expectedError: HaveOccurred(),
+			}),
+			Entry("no cluster name", parseClusterIdentityTestCase{
+				args:          []string{"--cluster-id=foo"},
+				expectedError: HaveOccurred(),
+			}),
```

Comment:
if the cluster name is required, we have to force the user to set it

1. in the template files the `if .values.discovery.config.clusterName` checks are not required
2. the chart should check if the name is set, and fail accordingly
3. liqoctl should either fail if the name is not set or provide a pseudo-randomic name if none is set